mypy/nodes.py

```diff
@@ -1732,10 +2036,22 @@ def __str__(self) -> str:
         a[-1] += ')'
         return '\n'.join(a)
 
+    def serialize(self, fullname: str) -> JsonDict:
+        data = {'.class': 'SymbolTable'}  # type: JsonDict
+        for key, value in self.items():
+            if key == '__builtins__':
```

Comment:
Does this get added somewhere? It would be correct to include `__builtins__`, though probably not very useful. Perhaps add a comment here.